.github/workflows/build.yml

```diff
@@ -138,6 +129,17 @@ jobs:
         run: |
           python -m pip install --upgrade pip setuptools wheel twine
           pip install -r requirements.txt
+          pip freeze > versions_at_release.txt
+      - name: Archive version artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: versions_at_release
+          path: |
+            versions_at_release.txt
+      - name: Create version environment variable
+        run: |
+          echo "GRASPOLOGIC_VERSION=`python setup.py --version`" >> $GITHUB_ENV
```

Comment:
Does this include information from the `egg_info -b "devBUILDNUMBER"` stuff too?

Reply:
Oh I guess it's only useful if you're on the main branch anyway

Reply:
Wonder if we should only conditionally run it then?

Reply:
No.  The BUILDNUMBER stuff is injected into documention in the Sphinx conf.py.  This is strictly the setup.cfg version and used only for the release/tag process.